mmv1/products/datastream/Stream.yaml

```diff
@@ -764,6 +784,211 @@ properties:
+                            item_type: !ruby/object:Api::Type::NestedObject
+                              description: |
+                                SQL Server Column.
+                              properties:
+                                - !ruby/object:Api::Type::String
+                                  name: 'column'
+                                  description: |
+                                    Column name.
+                                - !ruby/object:Api::Type::String
+                                  name: 'dataType'
+                                  description: |
+                                    The SQL Server data type. Full data types list can be found here:
+                                    https://learn.microsoft.com/en-us/sql/t-sql/data-types/data-types-transact-sql?view=sql-server-ver16
+                                - !ruby/object:Api::Type::Integer
+                                  name: 'length'
+                                  output: true
+                                  description: |
+                                    Column length.
+                                - !ruby/object:Api::Type::Integer
+                                  name: 'precision'
+                                  output: true
+                                  description: |
+                                    Column precision.
+                                - !ruby/object:Api::Type::Integer
+                                  name: 'scale'
+                                  output: true
+                                  description: |
+                                    Column scale.
+                                - !ruby/object:Api::Type::String
+                                  name: 'encoding'
+                                  output: true
+                                  description: |
+                                    Column encoding.
+                                - !ruby/object:Api::Type::Boolean
+                                  name: 'primaryKey'
+                                  output: true
+                                  description: |
+                                    Whether or not the column represents a primary key.
+                                - !ruby/object:Api::Type::Boolean
+                                  name: 'nullable'
+                                  output: true
+                                  description: |
+                                    Whether or not the column can accept a null value.
+                                - !ruby/object:Api::Type::Integer
+                                  name: 'ordinalPosition'
+                                  output: true
+                                  description: |
+                                    The ordinal position of the column in the table.
+          - !ruby/object:Api::Type::Integer
+            name: 'maxConcurrentCdcTasks'
+            send_empty_value: true
+            description: |
```

Comment:
The descriptions here & below don't quite match what's in the API docs: https://cloud.google.com/datastream/docs/reference/rest/v1/projects.locations.streams#SqlServerSourceConfig It seems like they're copy-pasted from previous blocks so I would believe they are likely the same but just wanted to double-check to see if there's other evidence you're aware of?

Reply:
You are correct. They were copied from the oracle source config. I don't have additional evidence. Updated in 869ce95a181c40ad0b0f2460f8880912a94e0266